app/scripts/modules/core/src/domain/IIndexed.ts

```diff
@@ -0,0 +1,3 @@
+export interface IIndexed {
```

Comment:
This was needed to make typescript happy in the abstract render class for the default cell renderer.  I don't love it.

Reply:
What about just using `get<string>(item, key)` in the cell renderers so we don't litter generic keys across all of our main types?